### PR TITLE
feat(3015866265): display the transfer button only if both wallets are connected

### PR DESCRIPTION
**Pull request by galuzan-starkware**
### Description of the Changes

display on the screen the `<TransferButton />` only if both wallets are connected, else will display `<LoginWalletButton />`.

### Checklist

- [ ] Changes have been done against master branch, and PR does not conflict
- [ ] New unit / functional tests have been added (whenever applicable)
- [ ] Test are passing in local environment
- [ ] Docs have been updated
- [ ] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/255)

### CODE REVIEW

**Comment by dan-ziv** [comment]
@galuzan-starkware the base branch should be `galu/feat/3015851363`

File: src/config/strings.js [hunk 1]
```diff
@@ -69,7 +69,8 @@ export default {
       maxBtnTxt: 'Max',
       balanceTitleTxt: 'Available balance',
       inputPlaceholderTxt: '0.00',
-      transferBtnTxt: 'Transfer'
+      transferBtnTxt: 'Transfer',
+      connectWalletBtnTxt: 'Connect Wallet'
```
Comment (dan-ziv) [inline src/config/strings.js:73]: If you read the component `LoginWalletButton` so stick with this prefix also here, or change the component name to `ConnectWalletButton`

**Review by dan-ziv — APPROVED** [review]

**Review by dan-ziv — CHANGES_REQUESTED** [review]
see my comment

File: src/components/UI/MainWalletButton/MainWalletButton.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {useColors} from '../../../hooks';
+import {toClasses} from '../../../utils';
+import {Button} from '../index';
+import styles from './MainWalletButton.module.scss';
+
+export const MainWalletButton = ({text, onClick, ...props}) => {
```
Comment (dan-ziv) [inline src/components/UI/MainWalletButton/MainWalletButton.js:9]: **Transfer**Button is a **Wallet**Button?

**Review by dan-ziv — CHANGES_REQUESTED** [review]

File: src/components/UI/LoginWalletsButton/LoginWalletsButton.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import {useTransferTranslation} from '../../../hooks';
+import {MainWalletButton} from '../index';
+
+export const LoginWalletsButton = props => {
```
Comment (dan-ziv) [inline src/components/UI/LoginWalletsButton/LoginWalletsButton.js:6]: Let's call it `ConnectWalletButton`.

File: src/components/UI/MainWalletButton/MainWalletButton.js (hunk 2 above)
Comment (galuzan-starkware) [reply src/components/UI/MainWalletButton/MainWalletButton.js:9]: good point, I'll fix this

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**